The musl libc header /usr/include/sched.h fails with the string.h
memory functions redefined as macros, thus enable the mem tracking
only if __GLIBC__ is defined as well.

--- src/lib/util/corealloc.h	2015-06-06 19:27:52.467562344 +0200
+++ src/lib/util/corealloc.h	2015-06-06 19:30:09.602554928 +0200
@@ -84,7 +84,7 @@
 //  ADDDITIONAL MACROS
 //**************************************************************************
 
-#ifndef NO_MEM_TRACKING
+#if !defined(NO_MEM_TRACKING) && defined(__GLIBC__)
 // re-route classic malloc-style allocations
 #undef malloc
 #undef calloc
